python/pyspark/sql/functions.py

```diff
@@ -4884,6 +4884,52 @@ def check_string_field(field, fieldName):  # type: ignore[no-untyped-def]
         return _invoke_function("window", time_col, windowDuration)
 
 
+def window_time(
```

Comment:
Oh, we should also add this manually into `python/docs/source/reference/pyspark.sql/functions.rst`

Reply:
done